test/beacon/Voucher.test.ts

```diff
@@ -488,6 +489,59 @@ describe('Voucher', function () {
                 expect(await voucher.getSponsoredAmount(dealId)).to.be.equal(0);
             });
 
+            it('Should match orders boost with partial sponsored amount', async () => {
+                const sponsoredValue = BigInt(datasetPrice + workerpoolPrice);
+                const nonSupporedValue = BigInt(appPrice); // app wont be eligible for sponsoring
```

Comment:
```suggestion
                const noSponsoredValue = BigInt(appPrice); // app wont be eligible for sponsoring
```
with other occurences.